version/version.go

```diff
@@ -55,6 +55,7 @@ type MeshInfo []ServerInfo
 type ProxyInfo struct {
 	ID           string
 	IstioVersion string
+	Type         string
```

Comment:
can we define a few types as enum?

Reply:
Just copied & pasted the `NodeType` from the istio repo.